pipelines/jobs/configurations/jdk17u_pipeline_config.groovy

```diff
@@ -74,14 +74,14 @@ class Config17 {
                 os                  : 'windows',
                 arch                : 'aarch64',
                 crossCompile        : 'x64',
-                buildArgs           : '--cross-compile',
                 additionalNodeLabels: 'win2016&&vs2019',
                 test                : [
                         nightly: [],
                         weekly : []
                 ],
-                buildArgs           : [
-                        "temurin"   : '--create-jre-image --create-sbom'
+                buildArgs       : [
```

Comment:
OpenJ9 doesn't build on Windows/aarch64 so that doesn't need to be included and may cause confusion, although equally doesn't cause any harm

Reply:
i can remove openj9 ,  no confusion for future change